lib/Basic/ObjectFile.cpp

```diff
@@ -0,0 +1,21 @@
+//===--- ObjectFile.cpp - Object File Related Information ------*- C++ -*-===//
+//
+// Object File related data structures.
+//
+//===----------------------------------------------------------------------===//
+
+#include "swift/ABI/ObjectFile.h"
+#include "llvm/Support/ErrorHandling.h"
+
+using namespace swift;
+
+SwiftObjectFileFormat &SwiftObjectFileFormat::getSwiftObjectFileFormat() {
```

Comment:
Are we sure that a static implementation is good enough for all use-cases? Is this function still useful if we are, for example on a Mac and debug a Linux process over the network?

Reply:
Ok! I changed this so the file format is resolved on LLDB's side, by consulting the module's triple's format type.

Reply:
Does that mean we can remove this function?